cliconfig/loader.go

```diff
@@ -392,18 +392,22 @@ func (l Loader) normalizeField(fieldName string, normalization string) error {
 		value, _ := reflections.GetField(l.Config, fieldName)
 		fieldKind, _ := reflections.GetFieldKind(l.Config, fieldName)
 
-		// Make sure we're normalizing a string filed
+		// Make sure we're normalizing a string field
 		if fieldKind != reflect.Slice {
 			return fmt.Errorf("list normalization only works on slice fields")
 		}
 
-		// Normalize the field to be a command
+		// Normalize the field to be a string
 		if valueAsSlice, ok := value.([]string); ok {
 			normalizedSlice := []string{}
 
 			for _, value := range valueAsSlice {
 				// Split values with commas into fields
 				for _, normalized := range strings.Split(value, ",") {
+					if normalized == "" {
```

Comment:
This change has broader implications beyond the redaction change as it affects the other list type config parameters. 

Those fields are:
- Tags
- TagsFromEC2MetaDataPaths
- TagsFromGCPMetaDataPaths
- Experiments

None of those convey semantic meaning to an empty string so I don’t have concerns that this will result in a change in practice.